### ISSUE TITLE
XYZ

### ISSUE THREAD

**Issue by cortner** [JuliaMolSim/JuLIP.jl#149]
Hi James - the ExtXYZ tests currently fail. Any idea? 
```
Test ExtXYZ fio for Atoms
extxyz: Error During Test at /Users/ortner/gits/JuLIP.jl/test/test_fio.jl:55
  Got exception outside of a @test
  MethodError: no method matching genperm(::Tuple{StaticArrays.SOneTo{9}}, ::Tuple{Int64, Int64})
  Closest candidates are:
    genperm(::Tuple{Vararg{Any, N}}, ::Tuple{Vararg{Int64, N}}) where N at ~/gits/julia17/usr/share/julia/base/permuteddimsarray.jl:80
    genperm(::Any, ::AbstractVector{Int64}) at ~/gits/julia17/usr/share/julia/base/permuteddimsarray.jl:81
  Stacktrace:
    [1] permutedims(A::StaticArrays.SVector{9, Float64}, perm::Tuple{Int64, Int64})
      @ Base.PermutedDimsArrays ./permuteddimsarray.jl:125
    [2] write_frame(fp::Ptr{Nothing}, dict::Dict{String, Any}; verbose::Bool)
      @ ExtXYZ ~/.julia/packages/ExtXYZ/6nS90/src/ExtXYZ.jl:418
    [3] write_frame
      @ ~/.julia/packages/ExtXYZ/6nS90/src/ExtXYZ.jl:416 [inlined]
    [4] (::ExtXYZ.var"#27#28"{Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}}, String, Vector{Dict{String, Any}}})(fp::Ptr{Nothing})
      @ ExtXYZ ~/.julia/packages/ExtXYZ/6nS90/src/ExtXYZ.jl:447
    [5] cfopen(f::ExtXYZ.var"#27#28"{Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}}, String, Vector{Dict{String, Any}}}, filename::String, mode::String)
      @ ExtXYZ ~/.julia/packages/ExtXYZ/6nS90/src/ExtXYZ.jl:59
```

**Comment by jameskermode** [comment]
Oops, not sure what has broken this. I’ll take a look

**Comment by cortner** [comment]
I didn't have time to do any extensive tests yet I'm afraid. Sorry.

**Comment by jameskermode** [comment]
It was broken by #148 so that shouldn't have been merged. Looks like CI is not running on pull requests, I'll try to figure out why at the same time as fixing this.

**Comment by jameskermode** [comment]
Actually, I might have spoken to soon and it might be a coincidence that [first failure](https://github.com/JuliaMolSim/JuLIP.jl/actions/runs/1808810545) was when #148 was merged. I'll investigate further.

**Comment by gelzinyte** [comment]
Definitely wasn't the intention. Typos aside, I don't think changes in #148 should be interacting with .xyz io?

**Comment by jameskermode** [comment]
Yes, I think you’re right, it was just a coincidence and something upstream has broken at the same time. I’ll keep digging.

**Comment by cortner** [comment]
closed by #152
